core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java

```diff
@@ -737,7 +737,6 @@ private TableUpdateDetails getTableUpdateDetailsFromSharedArea(
                         }
                     }
                     tableToken = engine.createTable(securityContext, ddlMem, path, true, tsa, false);
-                    LOG.info().$("created table [tableName=").$(tableNameUtf16).I$();
```

Comment:
is this intentionally removed?

Reply:
yes, it is redundant.

for OSS we already log this:
```
2023-10-25T22:12:01.986061Z I i.q.c.p.WriterPool created [table=`test~8`, thread=27]
```

for Ent we will have the same as above + EntDddlListener adds its own line which contains the username.